src/Beman/Optional26/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# cmake-format: off
+# src/Beman/Optional26/tests/CMakeLists.txt -*-makefile-*-
```

Comment:
Can we remove the above line?

Reply:
I suppose this is a duplicated thread of https://github.com/beman-project/Optional26/pull/29/files#r1678269909. Please reopen if I am wrong.